icu4c/source/test/cintltst/unumberrangeformattertst.c

```diff
@@ -67,7 +67,9 @@ static void TestExampleCode() {
     const UChar* str = ufmtval_getString(unumrf_resultAsValue(uresult, &ec), &len, &ec);
     assertSuccessCheck("There should not be a failure in the example code", &ec, TRUE);
     assertUEquals("Should produce expected string result", u"$3 – $5", str);
-    assertIntEquals("Length should be as expected", u_strlen(str), len);
+    if (ec == U_ZERO_ERROR) {
```

Comment:
We should not check for == zero-error.

Please either test `if (U_SUCCESS(ec)) {` or `if (str != nullptr) {`.

Alternatively, do
```
int32_t resultLength = str != nullptr ? u_strlen(str) : 0;
assertIntEquals("Length should be as expected", resultLength, len);
```

Reply:
Done.